README.md

```diff
@@ -26,6 +26,9 @@
       --host=<host>    If starting server, specify ip address to bind to.
                        Default is 127.0.0.1
       --port=<port>    Specify server port number. Default is 9288
+      --remote-trigger Defer printing until page evaluates the following:
+                       window.cefPdf({request: "", onSuccess: function () {}, onFailure: function () {}});
```

Comment:
Basically this is what is needed to be called in order to print the page the types need to add up but the values are basically arbitrary. Therefore

`{request: "", persistent: undefined, Success: function () {}, onFailure: function () {}};`

is the simplest argument which passes the type verification.

So the statement above (although perhaps complicated) is correct. Printing is deferred until the page evaluates `window.cefPdf({request: "", onSuccess: function () {}, onFailure: function () {}});`.

Something simpler could be implemented but that would involve reimplementing a lot of `include/wrapper/cef_message_router` and mean even more source to be maintained.

So instead I'm trying to find some way to add the function `function () { window.cefPdfImpl({request: "", onSuccess: function () {}, onFailure: function () {}}); }`  to `window` for convince.

Alternatively... and I'm not sure what I think about this, the same effect could be achieved using `CefDisplayHandler::OnConsoleMessage`. In that case the page would need to execute `console.log("cefPdf")` instead. This would make the implementation simpler for sure but it feels hacky somehow? What do you think? Using `CefDisplayHandler::OnConsoleMessage` may be the strongest solution despite this.